foxglove/console/client.go

```diff
@@ -72,7 +81,16 @@ func (c *FoxgloveClient) Stream(r *StreamRequest) (io.ReadCloser, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to serialize request: %w", err)
 	}
-	resp, err := c.authed.Post(c.baseurl+"/v1/data/stream", "application/json", buf)
+
+	req, err := http.NewRequest("POST", c.baseurl+"/v1/data/stream", buf)
+	if err != nil {
+		return nil, fmt.Errorf("failed to build request: %w", err)
+	}
+
+	req.Close = true
```

Comment:
why do we want to explicitly close here? is this to re-start the 1h connection timer on every stream request?

Reply:
I added this while trying to actually coax a usable error out of the failure case (one didn't materialize). The effect of this is to prevent the HTTP client from attempting to reuse the same TCP connection on followup requests. I should be able to remove it - shouldn't matter in practice here.